Makefile

```diff
@@ -1,5 +1,11 @@
 .PHONY: all lint test init pylint pep8 test-unit test-unit-via-clusterrunner test-functional freeze
 
+NAME=$(shell grep '^Name' .flute | grep -Eo '[[:alnum:]_-]+$$')
+RPM_VERSION_PREFIX=0.5
+COMMIT_COUNT=$(shell git rev-list --count HEAD)
```

Comment:
I'd rather we not reimplement the autoversioning logic in the Makefile. It should just use the ClusterRunner code. Can it not just execute `./main.py --version` to get the version?